scripts/monorepo/runPublished.js

```diff
@@ -0,0 +1,25 @@
+// @ts-check
+const getAllPackageInfo = require('./getAllPackageInfo');
+const runTo = require('./runTo');
+
+const argv = process.argv.slice(2);
+
+if (argv.length < 1) {
+  console.log(`Usage:
+
+  yarn run:published <script> [<args>]
+
+This command runs <script> for all beachball-published packages (and their dependencies).
+`);
+
+  process.exit(0);
+}
+
+const script = argv[0];
+const rest = argv.slice(1);
+
+const beachballPackages = Object.entries(getAllPackageInfo())
+  .filter(([, { packageJson, packagePath }]) => !/[\\/]fluentui[\\/]/.test(packagePath) && packageJson.private !== true)
```

Comment:
hmm, this feels less than ideal. we define `--scope` in `beachball:publish` command and we need to keep this in sync with that. what would happen if we didn't build the package but still let beachball to publish it?

Reply:
Not sure what you mean--the publish scope is just `!packages/fluentui/*` (and private packages are implicitly excluded), so this check should do the same thing.

Reply:
yes they are the same now. my concern is that we need to make sure they are in sync. what if we updated 
`beachball:publish` to have no scope while missing to update this code here?
with this being said, i don't really have a good suggestion other than adding comments